### PR TITLE
Tos Update

### PR DESCRIPTION
**Pull request by abziegler**

### CODE REVIEW

File: perma_web/perma/templates/terms_of_service.html [hunk 1]
```diff
@@ -38,15 +38,15 @@ <h1>Perma Terms Of Service</h1>
 
     <div class="col-sm-8 docs reading-body">
 
-      <h2 class="body-ah">Effective Date: September, 23, 2013</h2>
+      <h2 class="body-ah">Effective Date: January, 22, 2019</h2>
```
Comment (rebeccacremona) [inline perma_web/perma/templates/terms_of_service.html]: Is this formatting of a date, with a comma between the month and the day, a convention for this kind of doc? I see if was that way before. I'm more used to seeing, `January 22, 2019`

Comment (abziegler) [reply perma_web/perma/templates/terms_of_service.html]: That's known among experts as a "dumb typo" :) fixed

**Comment by bensteinberg** [comment]
It's not critical, but we might change `http:` to `https:` in four places.

File: perma_web/perma/templates/terms_of_service.html [hunk 2]
```diff
@@ -38,15 +38,15 @@ <h1>Perma Terms Of Service</h1>
 
     <div class="col-sm-8 docs reading-body">
 
-      <h2 class="body-ah">Effective Date: September, 23, 2013</h2>
+      <h2 class="body-ah">Effective Date: January, 22 2019</h2>
```
Comment (rebeccacremona) [inline perma_web/perma/templates/terms_of_service.html]: Wrong comma removed 😉

Comment (abziegler) [reply perma_web/perma/templates/terms_of_service.html]: And that's known as a "real dumber typo"

Comment (rebeccacremona) [reply perma_web/perma/templates/terms_of_service.html]: A certain song about Mondays comes to mind

**Pull request merged**